stages/org.osbuild.dmverity

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/python3
+import os
+import subprocess
+import sys
+import tempfile
+import osbuild.api
+
+
+def open_dmverity(blocksize, partlabel, datapart, hashpart, mountpoint):
+    output = subprocess.check_output(["/usr/sbin/veritysetup",
+                            "format", datapart, hashpart,
+                            "--data-block-size", f"{blocksize}"],
+                            universal_newlines=True)
+    for line in output.splitlines():
+        if line.startswith("Root hash:"):
+            root_hash = line.split(":", 1)[1].strip()
+            # Verify partition could be opened and mounted without any issues
+            subprocess.run(["/usr/sbin/veritysetup",
+                            "open", datapart, partlabel, hashpart, root_hash],
+                          check=True)
+            subprocess.run(["mount",
+                            "-o", "ro", f"/dev/mapper/{partlabel}", mountpoint],
+                          check=True)
+            return root_hash
+    raise ValueError(
+        f"cannot find expected 'Root hash' in veritysetup output:\n{output}")
```

Comment:
This function is a bit _weird_ in terms of name vs functionality.
For one, it's called `open_dmverity()` when the main thing it's doing is running `veritysetup format` and returning the root hash.  So I would, first of all, want this to be named something like `format_dmverity()` (or just `format()` since we're in the dmverity module anyway).

Now, I'm not convinced we need the verification step (`open`ing and `mount`ing the partition), but even if we do keep it, I'd make it a separate function completely.  However, I think mounting might cause issues.  If, for example, the `/dev/mapper/{partlabel}` already exists on the host, this will fail without good reason.  Whereas formatting will work without issue and the partition will be perfectly usable.

Reply:
Done. I dropped `mount`ing, but `open`ing is still here (i remember i had some issues with dm-verity on `s390x`)